libafl/src/executors/inprocess.rs

```diff
@@ -42,7 +42,7 @@ use crate::{
 };
 
 /// The process executor simply calls a target function, as mutable reference to a closure
-pub type InProcessExecutor<'a, H, HT, OT, S> = GenericInProcessExecutor<H, &'a mut H, HT, OT, S>;
+pub type InProcessExecutor<'a, H, OT, S> = GenericInProcessExecutor<H, &'a mut H, (), OT, S>;
```

Comment:
BTW, you wouldn't have needed the `GenericInProcessExecutor` indirection at all, you just had to implement `new()` only for:
```rust
impl<'a, H, OT, S> InProcessExecutor<'a, H, () OT, S> ...
```

Reply:
yes that makes sense.

Reply:
(GenericInProcessExecutor was here before my PR.)